tuning/automation/rocblas-benchInputCreator.py

```diff
@@ -215,38 +231,140 @@ def createYaml(args, outputfile, problem, sizeMappings, verify):
 
         sizeParams = getSizeParams(size, transA, transB)
 
+        latency = 2*sizeParams['M']*sizeParams['N']*sizeParams['K']/perf[1]/1000 # us
+        niters = math.ceil( 1.15 * 3e6 / latency)
+        
+        otherParams = {"alpha": 1, "beta": 1, "iters": niters, "cold_iters": niters}
+        otherParams_rotating = {"alpha": 1, "beta": 1, "iters": niters, "cold_iters": niters, "flush_memory_size": 536870812}
+        
+        niters=math.ceil( 1.25 * 3e6 / latency)
+        otherParams_beta0 = {"alpha": 1, "beta": 0, "iters": niters, "cold_iters": niters}
```

Comment:
Can we have an option to use default iteration (10, 2)?
I usually use this tool before converting Gflops into frequency.

Reply:
Or, use default value if perf is >100?

Reply:
If you run this script as before, it uses the default iters (10/2), but if you indicate -d duration, it updates the iterations.